apps/design/backend/src/language_and_audio/vendored_translations.json

```diff
@@ -265,7 +265,7 @@
     "Try the other input.": "Pruebe la otra entrada.",
     "Turn ballot over and continue voting": "Voltea la boleta y continúa votando",
     "Unmute Audio": "Activar audio",
-    "Unofficial TEST Ballot": "Boleta de PRUEBA no oficial",
+    "Unofficial Test Ballot": "Boleta de PRUEBA no oficial",
```

Comment:
As, missed this - should probably lowercase `PRUEBA` here also

Reply:
A got it, will do another PR